### PR TITLE
Added additional problems

### PR DESCRIPTION
**Pull request by dhayalramk**
Added additional problems which are unsolved or recent one.

### CODE REVIEW

**Comment by dhayalramk** [comment]
@vnbrs  Is it possible to merge the solutions of SQL!.

File: README.md [hunk 1]
```diff
@@ -5,8 +5,8 @@
 * [Advanced Select](https://github.com/vnbrs/hackerrank-sql/tree/master/advanced-select)
 * [Aggregation](https://github.com/vnbrs/hackerrank-sql/tree/master/aggregation)
 * [Basic Join](https://github.com/vnbrs/hackerrank-sql/tree/master/basic-join)
-* [Advanced Join](#) - WIP
-* [Alternative Queries](#) - WIP
+* [Advanced Join](https://github.com/vnbrs/hackerrank-sql/tree/master/advanced-join) - WIP
```
Comment (vinibrsl) [inline README.md]:
```suggestion
* [Advanced Join](https://github.com/vnbrs/hackerrank-sql/tree/master/advanced-join)
```

File: README.md [hunk 2]
```diff
@@ -5,8 +5,8 @@
...
+* [Alternative Queries](https://github.com/vnbrs/hackerrank-sql/tree/master/alternative-queries) - WIP
```
Comment (vinibrsl) [inline README.md:9]:
```suggestion
* [Alternative Queries](https://github.com/vnbrs/hackerrank-sql/tree/master/alternative-queries)
```

**Review by vinibrsl — CHANGES_REQUESTED** [review]

**Comment by vinibrsl** [comment]
Hi @dhayalramk. Thank you for your contribution.

I left 2 suggestions so we can merge your PR. Please accept them and then I'll merge it.

**Comment by dhayalramk** [comment]
@vnbrs  I accepted the suggestions  :) If anything pending my side, Please let me know.

**Comment by vinibrsl** [comment]
Merged! Thanks.

**Pull request merged**